submissions/asaMitaka/hooliStylePopup/style.css

```diff
@@ -0,0 +1,183 @@
+/* nav block */
+
+.nav__ul {
+    display: flex;
+    flex-direction: row;
+    justify-content: end;
+}
+
+.nav__item {
+    padding: 0 10px;
+    list-style: none;
+    display: flex;
+    align-items: center;
+}
+
+.nav__ul li a {
```

Comment:
Please, avoid styling by tags [link](https://frontstuff.io/you-need-to-stop-targeting-tags-in-css)